src/groups/mod.rs

```diff
@@ -347,7 +348,7 @@ impl<P: GroupParams> G<P> {
                 y: self.y,
             })
         } else {
-            let zinv = self.z.inverse_unconstrained().unwrap();
+            let zinv = self.z.inverse().unwrap();
```

Comment:
Why did you change this from `inverse_unconstrained`

Reply:
inverse_unconstrained is now deprecated